doc/userguide/upgrade.rst

```diff
@@ -60,6 +60,10 @@ Major changes
 - It is possible to see an increase of alerts, for the same rule-sets, if you
   use many stream/payload rules, due to Suricata triggering TCP stream
   reassembly earlier.
+- Datasets of the type String had a bug fixed which now takes into account the length of the string
```

Comment:
May I suggest alternate wording?

`Datasets of type String now include the length of the strings to determine if the memcap value is reached.`